typescript/libs/service_proxy.py

```diff
@@ -155,11 +156,19 @@ def quick_info(self, path, location=Location(1, 1), onCompleted=None):
         args = {"file": path, "line": location.line, "offset": location.offset}
         req_dict = self.create_req_dict("quickinfo", args)
         json_str = json_helpers.encode(req_dict)
-        self.__comm.sendCmd(
-            lambda json_dict: None if onCompleted is None else onCompleted(json_dict),
-            json_str,
-            req_dict["seq"]
-        )
+        callback = lambda json_dict: None if onCompleted is None else onCompleted(json_dict)
```

Comment:
I didn't quite understand last time you mentioned it. Now I got it :). That would be more concise.